apps/music/js/music.js

```diff
@@ -128,7 +134,13 @@ var App = (function() {
     titleElement.dataset.l10nId = l10nIds.title;
     textElement.dataset.l10nId = l10nIds.text;
 
-    document.getElementById('overlay').classList.remove('hidden');
+    Array.forEach(document.body.children, function(elt) {
+      if (elt.id === 'overlay') {
+        elt.classList.remove('hidden');
+      } else {
+        elt.setAttribute('aria-hidden', true);
+      }
+    });
```

Comment:
I'm not really sure if it is worth it. They are both in separate methods that are already there to show/hide the overlay, we might save 2-3 lines of code. I'll add comments above each of those loops.